src/models/transfer.py

```diff
@@ -218,3 +229,26 @@ def from_slippage(cls, slippage: SolverSlippage) -> Transfer:
             receiver=slippage.solver_address,
             amount_wei=slippage.amount_wei,
         )
+
+    @classmethod
+    def sort_list(cls, transfer_list: list[Transfer]) -> None:
```

Comment:
nit: Not a python expert, but I find it weird to have a class method that doesn't use any class state and returns nothing (because it sorts the list in place). Shouldn't this be a free method rather or return the sorted list?

Reply:
Yes actually this should be a static method. Whoops.